snap/seed_yaml_test.go

```diff
@@ -81,3 +81,20 @@ func (s *seedYamlTestSuite) TestNoPathAllowed(c *C) {
 	_, err = snap.ReadSeedYaml(fn)
 	c.Assert(err, ErrorMatches, `"foo/bar.snap" must be a filename, not a path`)
 }
+
+var nilSeedItemYaml = []byte(`
+snaps:
+ - name: foo
```

Comment:
I was actually thinking about replicating the odd format of seed.yaml in the images, `-` followed by a newline and actual properties in the subsequent lines.